### PR TITLE
Docs 8014 pm roles setup

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

File: modules/ROOT/pages/setup.adoc [hunk 1]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
 
 * Mule runtime engine (Mule) 4.2.2 or later (for On-Premises only).
 
-* The Partner Manager Administrator role for all users who need access to B2B message flows in their environment (Production or  Sandbox).
-
 * Keep the default deployment of message flows to CloudHub (for information on CloudHub settings, refer to xref:cloudhub-deploy-options.adoc[CloudHub Deployment Options], or deploy to your own hosted on-premise Mule runtime, by clicking *Stop deploying to CloudHub* and going to  xref:deploy-onpremise.adoc[Deploy On-Premises for Partner Manager]).
 
+* The Partner Manager Administrator role that assigns permission levels for all users who need access to B2B message flows in their environment (Production or  Sandbox).
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* The Partner Manager Administrator role that assigns permission levels for all users who need access to B2B message flows in their environment (production or sandbox)
```

File: modules/ROOT/pages/setup.adoc [hunk 2]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+=== Parter Manager Roles
+
+These levels of roles can be assigned to your Partner Manager users.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
You can assign your Partner Manager users the following roles:
```

File: modules/ROOT/pages/setup.adoc [hunk 3]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+*  Partner Manager Administrator has full permissions.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
*  Partner Manager Administrator 
Has full permissions
```

File: modules/ROOT/pages/setup.adoc [hunk 4]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+* View Host, Partners and Message Flows has view-only access to the host, partner, and message flow configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* View Host, Partners, and Message Flows
Has view-only access to the host, partner, and message flow configurations; cannot view transaction activity
```

File: modules/ROOT/pages/setup.adoc [hunk 5]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+Cannot view Transactions activity.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc:48]: See above.

File: modules/ROOT/pages/setup.adoc [hunk 6]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+* Manage Partners and Message Flows can create, modify, and delete partners or message flow configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* Manage Partners and Message Flows
Can create, modify, and delete partners or message flow configurations; can view host partner configurations but cannot view transaction activity
```

File: modules/ROOT/pages/setup.adoc [hunk 7]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+Can view the host partner configurations, but cannot view Transactions activity.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]: See above.

File: modules/ROOT/pages/setup.adoc [hunk 8]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+* Manage Activity can view and manage transaction activity, but cannot view or modify either partner or message flow configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc:55]:
```suggestion
* Manage Activity
Can view and manage transaction activity, but cannot view or modify either partner or message flow configurations
```

File: modules/ROOT/pages/setup.adoc [hunk 9]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+* Manage Host can create, modify, and delete host configurations,
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* Manage Host
Can create, modify, and delete host configurations
```

File: modules/ROOT/pages/setup.adoc [hunk 10]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+but cannot view or modify trading partner configurations or activity.
+This is also true for Partner Manager Settings under Access Management (when the user also has the Org Admin role).
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc:60]: What "is also true"? That the MH role can? cannot?

Comment (kbostrom) [reply modules/ROOT/pages/setup.adoc:60]: Where is Org Admin role described?

File: modules/ROOT/pages/setup.adoc [hunk 11]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
...
+
+* View Activity has view-only access to transaction activity, and cannot view or modify either partner or message flow configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* View Activity
Has view-only access to transaction activity and cannot view or modify either partner or message flow configurations.
```

File: modules/ROOT/pages/setup.adoc [hunk 12]
```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
 
 * Mule runtime engine (Mule) 4.2.2 or later (for On-Premises only).
 
-* The Partner Manager Administrator role for all users who need access to B2B message flows in their environment (Production or  Sandbox).
-
 * Keep the default deployment of message flows to CloudHub (for information on CloudHub settings, refer to xref:cloudhub-deploy-options.adoc[CloudHub Deployment Options], or deploy to your own hosted on-premise Mule runtime, by clicking *Stop deploying to CloudHub* and going to  xref:deploy-onpremise.adoc[Deploy On-Premises for Partner Manager]).
 
+* The Partner Manager Administrator role that assigns permission levels for all users who need access to B2B message flows in their environment (Production or  Sandbox).
+
+=== Parter Manager Roles
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc:37]: Is this text about naming and describing "roles" or "levels of access"?
Most of the following items do not look like role names, but look more like levels that should not be capitalized as if they were names.
If these are labels in the syntax, then they should be enclosed in ` `.

**Review by kbostrom — APPROVED** [review]
Edited

File: modules/ROOT/pages/setup.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 8 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc:55]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 9 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 11 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

**Pull request merged**